Sources/SafeDIMacros/Macros/InstantiableMacro.swift

```diff
@@ -365,3 +357,28 @@ public struct InstantiableMacro: MemberMacro {
         }
     }
 }
+
+// MARK: - TypeDescription
+
+extension TypeDescription {
+    fileprivate var isInstantiable: Bool {
+        self == .simple(name: "Instantiable")
+            || self == .nested(
+                name: "Instantiable",
+                parentType: .simple(name: "SafeDI")
+            )
+            || self == .attributed(
+                .simple(name: "Instantiable"),
+                specifier: nil,
+                attributes: ["retroactive"]
+            )
+            || self == .nested(
+                name: "Instantiable",
+                parentType: .attributed(
+                    .simple(name: "SafeDI"),
+                    specifier: nil,
+                    attributes: ["retroactive"]
+                )
+            )
+    }
```

Comment:
this clearly does not scale but it works well enough for now.

Reply:
Seems reasonable safe for now. If another change is needed for Swift 7 then something else could be done.